nfl_strat_analysis.Rmd

```diff
@@ -0,0 +1,1687 @@
+# Analysis on NFL Offensive Strategy: Did the New York Giants get lucky?
+
+Swapnav Deka
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+```{r, include=FALSE}
+library(RSQLite)
+library(dplyr)
+library(stringr)
+library(scales)
+library(ggplot2)
+library(knitr)
+library(grid)
+library(gridBase)
+```
+
+```{r,include=FALSE,results='hide'}
+NFL2010 <- read.csv("https://drive.google.com/open?id=1VCxo53CKGq1IMj2Pn2c7tel8F6pJSXtx")
+NFL2011 <- read.csv("https://drive.google.com/open?id=1b3iXgSrxkpNcjOCm9ukWWFXsW5jwCUDW")
+NFL2012 <- read.csv("https://drive.google.com/open?id=1vYpj25dLsoOow9DOrR4z8dBzwu4oNxNW")
+NFL2013 <- read.csv("https://drive.google.com/open?id=1xjouqScJMqM5qBju1xM6R0JrD_rGb_yV")
+teams <- read.csv("https://drive.google.com/open?id=1NzBXDkChZd41oBWnWo4PRGZlvETFtdbY")
+weather_table <- read.csv(paste0("https://drive.google.com/open?id=1LkJzZ72RzwrER3NpYEr8DdncQHiE7730"))
+merged <- read.csv("https://drive.google.com/open?id=1E0Ill7A_a3lHFBhgG0dAP4rQRCWb6g5Y")
+
+
+dcon <- dbConnect(SQLite(), dbname = "nfl.sqlite")
```

Comment:
Error in match.names(clabs, names(xi)) : 
  names do not match previous names

Do you know why it raises error at this line?